code/modules/clothing/outfits/factions/syndicate.dm

```diff
@@ -594,7 +603,8 @@
 	suit = /obj/item/clothing/suit/armor/ngr/lieutenant
 	id = /obj/item/card/id/syndicate_command/crew_id
 	shoes = /obj/item/clothing/shoes/combat
-	suit_store = /obj/item/gun/ballistic/automatic/pistol/ringneck
+	suit_store = /obj/item/gun/ballistic/automatic/pistol/asp
```

Comment:
same here ig

Reply:
The Ensign one was actually a mistake because I started by copying the HoS datum, but the HoS spawns with a gun because he already spawned with a gun before I touched this, and so does every other syndicate HoS. I could remove it and shuffle the Asp to his locker, I suppose, but I won't change the Hardliner one or the base one. that'll have to wait until imaginos maps the hardliner carina.